ggdp/assets.py

```diff
@@ -171,10 +173,17 @@ def raw_allo_deployments() -> pd.DataFrame:
     return ipfs_content
 
 
-@asset(
-    compute_kind="Covalent_API",
-    group_name="chain_data",
-)
+@asset(compute_kind="API", group_name="private_api")
+def dune_allo_deployments(
+    dune: DuneResource, raw_allo_deployments: pd.DataFrame
+) -> None:
+    """
+    Uploads allo deployments to Dune.
+    """
+    dune.upload_csv(raw_allo_deployments, "allo_contract_deployments")
```

Comment:
This could potentially be a multi-asset that loops over assets tagges with `curated` or another tag.